### DISCUSSION TITLE
Heads up - Translations will need updating for slug and title changes (CSS)

### DISCUSSION THREAD

**Discussion by dipikabh** [mdn/mdn-community#388] [Localization]
Hello all,

I am updating the slugs and titles in the CSS module landing pages (`page-type: css-module`).
Some of the files in the child folders do have `page-type: guide` and they've been updated as well for consistency.

The translated content will need to be updated for these changes.

Multiple PRs in mdn/content are lined up for this:

- https://github.com/mdn/content/pull/26939
- https://github.com/mdn/content/pull/26958
- https://github.com/mdn/content/pull/26974

I wanted to bring this to your attention so that after these PRs land, you can update the translations when you have a moment.

Thank you for your contributions for keeping the translated content space up to date!! 🙏

**Comment by yin1999** [comment]
Thanks for the note. After https://github.com/mdn/yari/pull/7886 is landed, the bot in translated-content could automatically sync slug changes in case-sensitive mode (see also: https://github.com/mdn/translated-content/pull/13401). So, we just need to force on titles, referenced URLs, and other content :)

**Comment by pransh15** [comment]
Closing this as it has been resolved.
